clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -91,11 +93,26 @@ def start_clipper(
             compability and preserve the expected behavior of the system.
         cache_size : int, optional
             The size of Clipper's prediction cache in bytes. Default cache size is 32 MiB.
-
+        config_file: str(optional)
+            File for the configuration of the application or model
         Raises
         ------
         :py:exc:`clipper.ClipperException`
         """
+        if(config_file is not None):
```

Comment:
This logic should occur after clipper has been started successfully. Move it to a separate method called `parse_configuration` and invoke the method after the try/catch block below.